use tflite bilinear op to resize input of label_image

freedomtan opened the pull request:
replace previous naive `downsize()` function with a `resize()` using TF Lite RESIZE_BILINEAR operator

freedomtan committed:
use bilinear op to resize input of label_image

replace previous naive downsize() function with TF Lite
RESIZE_BILINEAR operator

andrehentz commented on tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h:
```diff
@@ -16,30 +16,74 @@ limitations under the License.
+  // fill input image
+  // in[] are integers, cannot do memcpy() directly
+  auto input = interpreter->typed_tensor<float>(0);
+  for (int i = 0; i < number_of_pixels; i++) input[i] = in[i];
```
please add curly braces and use multiple lines.

freedomtan replied:
OK, will do. Clang-format did it :-)

andrehentz commented on tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h:
```diff
@@ -16,30 +16,74 @@ limitations under the License.
+            int wanted_channels, Settings* s) {
+
+  int number_of_pixels = image_height * image_width * image_channels;
+  std::unique_ptr<Interpreter> interpreter(new Interpreter);
```
This is a clever idea. I wish we had a more general way to manipulate the original graph and add the extra node directly. That would avoid the need to instantiate a separate interpreter just to resize the image. Anyway, no action required. This is just food for thought.

freedomtan replied:
This idea is actually stolen from [ReadTensorFromImageFile()](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/main.cc#L114) of the original `tensorflow/examples/label_image/` :-)

rmlarsen commented:
@freedomtan Please address @andrehentz 's comments.

freedomtan committed:
add brackets and new line

Add curly brackets and new line to address the format problem
mentioned in review.
